pyproject.toml

```diff
@@ -0,0 +1,163 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "EveryVoice"
+description = "Text-to-Speech Synthesis for the Speech Generation for Indigenous Language Education Small Teams Project"
+license = { file = "LICENSE" }
+authors = [
+  { name = "Aidan Pine", email = "Aidan.Pine@nrc-cnrc.gc.ca" },
+  { name = "Eric Joanis", email = "Eric.Joanis@nrc-cnrc.gc.ca" },
+  { name = "Marc Tessier", email = "Marc.Tessier@nrc-cnrc.gc.ca" },
+  { name = "Mengzhe Geng", email = "Mengzhe.Geng@nrc-cnrc.gc.ca" },
+  { name = "Samuel Larkin", email = "Samuel.Larkin@nrc-cnrc.gc.ca" },
+]
+maintainers = [
+  { name = "Aidan Pine", email = "Aidan.Pine@nrc-cnrc.gc.ca" },
+  { name = "Eric Joanis", email = "Eric.Joanis@nrc-cnrc.gc.ca" },
+  { name = "Marc Tessier", email = "Marc.Tessier@nrc-cnrc.gc.ca" },
+  { name = "Mengzhe Geng", email = "Mengzhe.Geng@nrc-cnrc.gc.ca" },
+  { name = "Samuel Larkin", email = "Samuel.Larkin@nrc-cnrc.gc.ca" },
+]
+readme = "README.md"
+requires-python = ">=3.10, <3.12"
+keywords = ["TTS", "CLI"]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Console",
+  "Environment :: MacOS X",
+  "Environment :: Other Environment",
+  "Environment :: Win32 (MS Windows)",
```

Comment:
I would remove this. While I keep it more or less working on Windows for testing and dev purposes, it's quite broken on Windows in terms of the real functionality. Most importantly, no sox effects. So I would not make this claim.